CMakeLists.txt

```diff
@@ -158,6 +158,11 @@ if (FMT_PEDANTIC)
   target_compile_options(fmt PRIVATE ${PEDANTIC_COMPILE_FLAGS})
 endif ()
 
+if (NOT ${CMAKE_VERSION} VERSION_LESS 3.8)
+  target_compile_features(fmt INTERFACE cxx_std_11)
+endif ()
+
+
```

Comment:
nit: extra newline

Reply:
Will fix after confirmation of the other potential change